cypress/e2e/dev.spec.ts

```diff
@@ -34,15 +30,15 @@ describe('example run-util test', () => {
 // describe('example create users/truncate test', () => {
 //   it('can create users and truncate the database afterwards', () => {
 //     cy.truncateDatabase();
-//     cy.visit('http://localhost:3000/directory');
+//     cy.visit('/directory');
 //     cy.get('ul li').should('have.length', 0);
 //
 //     cy.createUsers(20);
-//     cy.visit('http://localhost:3000/directory');
+//     cy.visit('/directory');
 //     cy.get('ul li').should('have.length', 20);
 //
 //     cy.truncateDatabase();
-//     cy.visit('http://localhost:3000/directory');
+//     cy.visit('/directory');
```

Comment:
Do we still need this if it's commented out?

Reply:
@devonzara added this with the comment:

> // Example test would be too closely tied to the temporary UI
> // Left here for reference, but should remain commented.

Leaving it in for him to decide whether it should stay or not